server/models/user.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type User struct { 
+	ID        uuid.UUID `uuid.UUID gorm:"type:uuid;primary_key;"`
+	Name      string    `json:"name" binding:"required"`
+	Email     string    `json:"email" gorm:"unique" binding:"required"`
+	Password  string    `json:"password" binding:"required"`
+	Voucher   string    `json:"voucher"`
+	CreatedAt time.Time `json:"timestamp"`
+	Code      int       `json:"code"`
```

Comment:
why setting a code here?

Reply:
to compare it with the one cached and if same create acc for user